README.adoc

```diff
@@ -3239,7 +3239,7 @@ Person = Struct.new(:first_name, :last_name) do
 end
 ----
 
-=== No Extend `Struct.new` [[no-extend-struct-new]]
+=== Don't Extend `Struct.new` [[dont-extend-struct-new]]
```

Comment:
Please, check that there's no reference to this in RuboCop, as the anchor might need to be updated there as well.

Reply:
There was indeed, so I've changed it back.